### ISSUE TITLE
Projector Mode query parameter?

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/chipper#693]
In https://github.com/phetsims/chipper/blob/master/js/initialize-globals.js I see the following

```

    /**
     * The color profile used at startup, relevant only for sims that support multiple color profiles.
     * Such sims are required to have a 'default' profile.  If a sim supports a 'project mode' then
     * it should also have a 'projector' profile.  Other profile names are not currently standardized.
     */
    colorProfile: {
      type: 'string',
      defaultValue: 'default'
    },
```

What would the query parameter for projector mode look like in this case, would it be `?colorProfile=projector` ?

We are going to start revealing some query parameters in teacher tips, and this might be a nice one for teachers to have at their disposable so they can just load the sim in that mode if desired

**Comment by jonathanolson** [comment]
> What would the query parameter for projector mode look like in this case, would it be ?colorProfile=projector ?

Yes, that works on molecule-shapes. We'd probably want to check all of our color profiles.

**Comment by pixelzoom** [comment]
Related note, probably worth of its own issue... In a previous dev meeting, we noted that we should document each query parameter as either "internal use only" or "public facing".   That apparently still needs to be done for initialize-globals.js, and probably for most sim-specific query parameters.

**Comment by samreid** [comment]
See also https://github.com/phetsims/joist/issues/342 for discussion about color profile support.

**Comment by jbphet** [comment]
I looked through the code and found eight sims that support projector mode.  I tested the query parameter (`colorProfile=projector`) on the published and master versions.  Here are the results:

Sim | Works on Published Version | Works on Master Version
--- | --- | --- 
atomic-interactions | no | no
charges-and-fields | no | yes 
coulombs-law | n/a (unpublished) | yes 
gravity-and-orbits| yes| yes
molecule-shapes| no | yes
rutherford-scattering| no | no (crashes sim)
states-of-matter| yes | yes
states-of-matter-basics | yes | yes

I've created GitHub issues for the two that don't work on master, see https://github.com/phetsims/atomic-interactions/issues/71 and https://github.com/phetsims/rutherford-scattering/issues/141.  The former is assigned to me, the latter to @jessegreenberg.  Once these are fixed, we should have support for all sims on master.

Assigning back to @ariel-phet to decide on next steps.  If he was thinking about sharing this query param with teachers he'll need to take the state of the published sims into consideration.
